__tests__/payment-request/payment-request.spec.ts

```diff
@@ -19,62 +17,93 @@ const btcAmountInvoice =
 const mockOnChainAddress = "tb1qw508d6qejxtdg4y5r3zarvary0c5xw7kxpjzsx"
 
-const mockLnInvoice = createMock<LnInvoice>({
+// Manually created mock objects
+const mockLnInvoice = {
+  __typename: "LnInvoice",
   paymentRequest: btcAmountInvoice,
-})
-
-const mockLnInvoiceCreate = jest.fn().mockResolvedValue({
-  data: {
-    lnInvoiceCreate: {
-      invoice: mockLnInvoice,
-      errors: [],
-    },
-  },
-  errors: [],
-})
+  // Add other necessary properties here
+}
 
-const mockLnUsdInvoice = createMock<LnInvoice>({
+const mockLnUsdInvoice = {
+  __typename: "LnInvoice",
   paymentRequest: usdAmountInvoice,
-})
-
-const mockLnUsdInvoiceCreate = jest.fn().mockResolvedValue({
-  data: {
-    lnUsdInvoiceCreate: {
-      invoice: mockLnUsdInvoice,
-      errors: [],
-    },
-  },
-  errors: [],
-})
+  // Add other necessary properties here
+}
 
-const mockLnNoAmountInvoice = createMock<LnInvoice>({
+const mockLnNoAmountInvoice = {
+  __typename: "LnInvoice",
   paymentRequest: noAmountInvoice,
-})
+  // Add other necessary properties here
+}
 
-const mockLnNoAmountInvoiceCreate = jest.fn().mockResolvedValue({
-  data: {
-    lnNoAmountInvoiceCreate: {
-      invoice: mockLnNoAmountInvoice,
-      errors: [],
+const mockLnInvoiceCreate = jest.fn(() =>
+  Promise.resolve({
+    data: {
+      __typename: "Mutation", // Correct placement according to your schema
+      lnInvoiceCreate: {
+        __typename: "LnInvoicePayload",
+        invoice: mockLnInvoice,
+        errors: [],
+      },
     },
-  },
-  errors: [],
-})
-
-const mockOnChainAddressCurrent = jest.fn().mockResolvedValue({
-  data: {
-    onChainAddressCurrent: {
-      address: mockOnChainAddress,
-      errors: [],
+    errors: [],
+  }),
+)
+
+const mockLnUsdInvoiceCreate = jest.fn(() =>
+  Promise.resolve({
+    data: {
+      __typename: "Mutation", // Correct placement according to your schema
+      lnUsdInvoiceCreate: {
+        __typename: "LnInvoicePayload",
+        invoice: mockLnUsdInvoice,
+        errors: [],
+      },
     },
-  },
-  errors: [],
-})
+    errors: [],
+  }),
+)
+
+const mockLnNoAmountInvoiceCreate = jest.fn(() =>
+  Promise.resolve({
+    data: {
+      __typename: "Mutation", // Correct placement according to your schema
+      lnNoAmountInvoiceCreate: {
+        __typename: "LnNoAmountInvoicePayload",
+        invoice: mockLnNoAmountInvoice,
+        errors: [],
+      },
+    },
+    errors: [],
+  }),
+)
+
+const mockOnChainAddressCurrent = jest.fn(() =>
+  Promise.resolve({
+    data: {
+      __typename: "Mutation", // Correct placement according to your schema
+      onChainAddressCurrent: {
+        __typename: "OnChainAddressPayload",
+        address: mockOnChainAddress,
+        errors: [],
+      },
+    },
+    errors: [],
+  }),
+)
 
 export const mutations: GeneratePaymentRequestMutations = {
+  // eslint-disable-next-line
+  // @ts-ignore
```

Comment:
Why is ts-ignore rquired here? I think if we're going to add ts-ignore there should be a comment that explains why it is neccessary.